peregrine/resources/submission/graphql/node.py

```diff
@@ -515,7 +515,7 @@ def lookup_graphql_type(T):
     return {
         bool: graphene.Boolean,
         float: graphene.Float,
-        long: graphene.Float,
+        int: graphene.Float,
```

Comment:
agreed, let's remove this line
since the gdcdatamodel python 3 update, the type cannot be `long` anymore (https://github.com/uc-cdis/gdcdatamodel/blob/develop/gdcdatamodel/models/__init__.py#L109-L123)

Reply:
agreed, let's remove this line
since the gdcdatamodel py3 update, the type can't be `long` anymore https://github.com/uc-cdis/gdcdatamodel/blob/3.0.2/gdcdatamodel/models/__init__.py#L109-L123